### ISSUE TITLE
Update Darknet to perform classification using models like darknet19

### ISSUE THREAD

**Issue by jrobble** [openmpf/openmpf#1044]
The darknet19 model is able to perform classification using the 1000 ImageNet classes. The output of the model is a SOFTMAX layer that returns a collection of 1000 floating point confidence values (similar to the GoogLeNet model, which used to be part of the Caffe component, now OcvDnnDetection component). This is different from the output of the YOLO models which return regions. Thus, to support the darknet19 model, the Darknet component must be updated to work with different outputs from the activated network.

**Comment by lmtoothman** [comment]
Can we make this more generic - I'm not tied to the Darknet19 model (I was just using it as an example)? I'm assuming that once the component is updated to handle the 1000 floating point values as well, any of the "Pre-Trained Models" listed here should work: https://pjreddie.com/darknet/imagenet/

**Comment by jrobble** [comment]
@lmtoothman , we'll take a look at the other models too and see if they can be handled in the same way.

**Comment by clnowacki** [comment]
The Darknet component has been archived.
